packages/manager/apps/pci-object-storage/src/pages/objects/container/object/show/ContainerContext.tsx

```diff
@@ -0,0 +1,102 @@
+import { createContext, useContext } from 'react';
```

Comment:
This file should be entirely reworked, as I'm not sure you need to declare all of these contexts